weblate/templates/trans/alert/nomaskmatches.html

```diff
@@ -4,6 +4,6 @@
 <p>{% trans "There are no translations in this component." %}</p>
```

Comment:
```suggestion
<p>{% trans "This component has no translations." %}</p>
```
Yet? Don't know how this is used.